### PR TITLE
[MBUILDCACHE-60] Remove undeclared Guava dependency with Arrays.asList

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Comment by elharo** [comment]
ping

**Comment by olamy** [comment]
> ping

why ping?
if CI is green just merge it. no need to ask review for such a minor change because it's just modifying test sources not sure you need to ping people to validate this.

**Comment by elharo** [comment]
I can't agree with that. Code review is essential. If it's that trivial, then it's an easy review. If it isn't, it needs review all the more. I've certainly noticed more than one mistake in Maven when code review was skipped, both by myself and others.

File: src/test/java/org/apache/maven/buildcache/LifecyclePhasesHelperTest.java
Comment (ferdnyc) [inline src/test/java/org/apache/maven/buildcache/LifecyclePhasesHelperTest.java:21]: I notice you left the `ArrayList` import here (as opposed to in the other files), is that still needed?

**Review by ferdnyc — COMMENTED** [review]
Looks sensible enough to me.

**Comment by ferdnyc** [comment]
I'm with @elharo — I _hate_ merging my own code to a public repo, and will generally only do it if it's a non-code change or I can definitively state that merging won't impact other users.

Otherwise, I figure a second pair of eyes is a sensible precaution on almost **any** code change, and when there are multiple people with commit access a good general rule is to have someone other than the PR author do the merge. I try to review others' changes as much as I can, in the hopes they'll return the favor.

In that spirit, I've pitched in with an inexpert review of the changes here.

File: src/test/java/org/apache/maven/buildcache/LifecyclePhasesHelperTest.java
Comment (elharo) [reply src/test/java/org/apache/maven/buildcache/LifecyclePhasesHelperTest.java:21]: Yes, it's used in a couple of unchanged lines in this class

**Review by gnodet — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #331

**Comment by jira-importer** [comment]
Resolve #331

**Pull request merged**
